cmd/web.go

```diff
@@ -67,6 +67,7 @@ func runHTTPRedirector() {
 		if len(r.URL.RawQuery) > 0 {
 			target += "?" + r.URL.RawQuery
 		}
+		// FIXME: Why aren't we adding the Fragment?
```

Comment:
The answer is: fragment should not be sent to server.

https://www.rfc-editor.org/rfc/rfc7230.html#section-5.3

But there is a HTTP header to pass the fragment:

https://www.w3.org/2008/WebVideo/Fragments/wiki/Server-parsed_Fragments

Reply:
Cool.

I thought that was the case but thought it better to ask.

```suggestion
```